source/SemVerAnalyzer/Nuget/NugetClient.cs

```diff
@@ -21,7 +25,32 @@ public async Task<byte[]> GetAssemblyBytesFromPackage(string packageName, string
 			try
 			{
 				using var client = new HttpClient();
-				using var request = new HttpRequestMessage(HttpMethod.Get, Path.Join(_config.RepositoryUrl, packageName));
+				
+				using var feedRequest = new HttpRequestMessage(HttpMethod.Get, Path.Combine(_config.RepositoryUrl, "v3", "index.json"));
+				using var feedResponse = await client.SendAsync(feedRequest);
+				if (!feedResponse.IsSuccessStatusCode)
+				{
+					var content = await feedResponse.Content.ReadAsStringAsync();
+					comments.Add($"Error retrieving Nuget feed:\n{content}");
+					return null;
+				}
+
+				var feed = JsonSerializer.Deserialize<NugetPackageFeed>(await feedResponse.Content.ReadAsStringAsync());
+				var packageBaseAddress = feed.Resources.Single(r => r.Type.StartsWith("PackageBaseAddress")).Id;
```

Comment:
Is this sufficient?  E.g. base = `Microsoft.Owin` when there exists that plus `Microsoft.Owin.xxx` as well...

Reply:
Agreed.  Also, in the case of multiple packages starting with the same text (which is likely for _any_ package that might use this tool), `.Single()` would throw.

Reply:
As far as I'm aware yes - I'm not sure what other suffixes there could be - it depends on the nuget API implementation. For our purposes however, this is sufficient - there are no other like strings

Reply:
Would we not want it to throw? I'd want to be notified as early as possible if there were multiple types like this, and my assumptions were incorrect

Reply:
Also @gregsdennis this url doesn't provide the information you're thinking. Please check https://api.nuget.org/v3/index.json for reference